x-pack/plugins/observability_solution/slo/public/embeddable/slo/alerts/components/slo_alerts_table.tsx

```diff
@@ -14,7 +14,7 @@ import type { SloItem } from '../types';
 import { SLO_ALERTS_TABLE_CONFIG_ID } from '../../constants';
 
 const ALERTS_PER_PAGE = 10;
-const ALERTS_TABLE_ID = 'xpack.observability.sloAlertsEmbeddable.alert.table';
```

Comment:
@XavierM I am moving slo stuff in a new plugin as part of this PR and I just realized this was not a translation copy, but the alert table id. I am thinking to simply undo this change and use the old alerts table id.  Would changing the alert table id be an issue?

Reply:
Yes I will use the old alerts table id and make sure that this id is used to register the alert table configuration somewhere.